server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -87,8 +92,7 @@ pub struct MessageOut {
     #[serde(rename = "eventId")]
     pub uid: Option<MessageUid>,
     pub event_type: EventTypeName,
-    pub payload: serde_json::Value,
-
+    pub payload: Option<serde_json::Value>,
```

Comment:
This change breaks the API (and compatibility with prod). Can just return some sort of a constant e.g. `{}` or better yet: `{"removed": true}`